Unit Tests and Integration tests broked

### Expected Behavior
Unit tests and Integration tests broken

### Current Behavior
<!--- Tell us what happens instead of the expected behavior -->
Unit tests and integration tests fail.

### Possible Solution
<!--- Not obligatory, but suggest a fix/reason for the bug, -->
Fix python packages to a version that worked.
Updating source to crypt4gh new version.

### Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
Unit Tests:
1. tox -r in root directory
```
=========================================================================================== ERRORS ===========================================================================================
_________________________________________________________________________ ERROR collecting tests/unit/test_ingest.py _________________________________________________________________________
ImportError while importing test module '/home/stenegru/csc/dev/neic-LocalEGA/tests/unit/test_ingest.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/unit/test_ingest.py:2: in <module>
    from lega.ingest import main, work
lega/ingest.py:24: in <module>
    from legacryptor.crypt4gh import get_header
.tox/unit_tests/lib/python3.7/site-packages/legacryptor/crypt4gh.py:16: in <module>
    import pgpy # Simply used to encrypt/decrypt the records, all in memory.
.tox/unit_tests/lib/python3.7/site-packages/pgpy/__init__.py:5: in <module>
    from .pgp import PGPKey
.tox/unit_tests/lib/python3.7/site-packages/pgpy/pgp.py:45: in <module>
    from .packet import Key
.tox/unit_tests/lib/python3.7/site-packages/pgpy/packet/__init__.py:9: in <module>
    from .packets import *  # NOQA
.tox/unit_tests/lib/python3.7/site-packages/pgpy/packet/packets.py:18: in <module>
    from .fields import DSAPriv, DSAPub, DSASignature
.tox/unit_tests/lib/python3.7/site-packages/pgpy/packet/fields.py:30: in <module>
    from cryptography.hazmat.primitives.asymmetric import ed25519
E   ImportError: cannot import name 'ed25519' from 'cryptography.hazmat.primitives.asymmetric' (/home/stenegru/csc/dev/neic-LocalEGA/.tox/unit_tests/lib/python3.7/site-packages/cryptography/hazmat/primitives/asymmetric/__init__.py)
====================================================================================== 1 error in 0.52s ======================================================================================
ERROR: InvocationError for command '/home/stenegru/csc/dev/neic-LocalEGA/.tox/unit_tests/bin/coverage run -m pytest -x tests/unit' (exited with code 1)
```

Integration tests:
1. `bats integration` after the system has been bootstraped - see https://travis-ci.org/neicnordic/LocalEGA/jobs/591996474

### Context (Environment)
<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

<!--- Provide a general summary of the issue in the Title above -->
* Travis
* local Development
### Related:
https://github.com/EGA-archive/LocalEGA/issues/78

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,7 +1,7 @@
 pika==1.0.1
 aiohttp==3.3.2
 sphinx_rtd_theme
-cryptography==2.3.1
+cryptography==2.6.1
 pgpy
 psycopg2-binary==2.7.5
 PyYaml

